### ISSUE TITLE
Bootstrap community.kubevirt setup

### ISSUE THREAD

**Issue by gundalow** [ansible-collections/community.kubevirt#3]
These modules were in [community.general](https://github.com/ansible-collections/community.general), we've span them out into their own collection/.

Context https://github.com/ansible/community/issues/539#issuecomment-720085434

Tracking: https://github.com/ansible-collections/overview/issues/128#issuecomment-730620626

- [x] `sanity` test in GitHub Action green
- [x] `unit` test in GitHub Action green
- [x] `integration` test in GitHub Action green
- [x] Review against [collection_requirements](https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst) (done and added missing items to checklist)
- [x] Codecoverage setup
- [x] Setup Zuul for build+publish of collection https://github.com/ansible/project-config/
- [ ] Update [documentation overview](https://hackmd.io/uKhHRhFUStG_bVNhY10cLQ)
- [ ] ~~Should Python 2.6 be removed from `collection_template`~~ Py 2.6 is still supported on remote
- [x] ~~Speak with Matt about Python virtualenv vs [default-container](https://github.com/ansible/default-test-container/blob/master/Dockerfile#L26-L35)~~ https://github.com/ansible/ansible/issues/72738
- [x] [Change default branch to "main"](https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst#branch-name-and-configuration)
- [x] [Set up branch protections](https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst#branch-name-and-configuration) (can they be added automatically from the template or otherwise?)
- [x] Check for missing commits from community.general (do file level diff against all file in both repos)
- [ ] action groups https://github.com/ansible/ansible/pull/72496
- [ ] removed from community.general

PR: https://github.com/ansible-collections/community.kubevirt/pull/4

Branch `bootstrap-collection`

### Removal from c.g

- [ ] `tests/unit/requirements.txt` openshift ; python_version >= '2.7'

**Comment by gundalow** [comment]
Installing Galaxy dependencies https://github.com/ansible-collections/community.routeros/blob/main/.github/workflows/ansible-test.yml#L90-L91

**Comment by dmsimard** [comment]
PRs for zuul project-config:
- https://github.com/ansible/project-config/pull/659
- https://github.com/ansible/project-config/pull/660

**Comment by dmsimard** [comment]
PR for changing module/plugin names to FQCNs: https://github.com/ansible-collections/community.kubevirt/pull/5

**Comment by dmsimard** [comment]
We're missing commits from community.general because we bootstrapped the collection on top of an old checkout -- let's not do that again :)

These are the commits I found:
- [x] https://github.com/ansible-collections/community.general/commit/22c04d54f2690237092d0ef88b7555f152a7b772
- [x] https://github.com/ansible-collections/community.general/commit/f7656ac0d3a82845dd12accc804cbe0a66d8b9c1 (already done in our tree)
- [x] https://github.com/ansible-collections/community.general/commit/097aebadb09965a5d2f302557ba21e21d70764a1
- [x] https://github.com/ansible-collections/community.general/commit/41cfdda6a3cdd952645b40d4c9da817375e878c2 (already done in our tree)

Edit: sent a PR to add the two missing commits I found https://github.com/ansible-collections/community.kubevirt/pull/7

**Comment by dmsimard** [comment]
We're also going to need to add community.kubevirt to ansible/ansible like this so the module defaults can carry over https://github.com/ansible/ansible/pull/72496

**Comment by dmsimard** [comment]
WIP PR to remove from community.general: https://github.com/ansible-collections/community.general/pull/1317

**Comment by gundalow** [comment]
Anything left here before we can release `1.0.0`?

Overall tracking of collection migration can be found at https://github.com/ansible-collections/overview/issues/128#issuecomment-730620626

**Comment by dmsimard** [comment]
@gundalow as discussed elsewhere, I was hoping to at least manually test 0.1.0 against a kubevirt instance to make sure that they are not horribly broken before releasing 1.0.0. I reached out to Tim in order to find out if he can help with this.
